solr/solrj/src/java/org/apache/solr/common/cloud/acl/ZkCredentialsProvider.java

```diff
@@ -14,13 +14,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.solr.common.cloud;
+package org.apache.solr.common.cloud.acl;
```

Comment:
I think I missed the part of the discussion where we described why it is necessary to repackage? I suspect this is one of the major sources of incompatibility that would prevent a back port to the 9x release branch.